src/schema/objects/metadata.yaml

```
@@ -1115,6 +1163,62 @@ EthicsApprovals:
   type: array
   items:
     type: string
+EventIdentifier:
+  name: EventIdentifier
+  display_name: Event Identifier
+  description: |
+    The message sent to the eye-tracker which disambiguates the belonging
+    of the sample to a certain event, condition or group.
+    This can be a unique trial identifier, or in the case of
+    continuous presentation (for example video) the annotated event of interest.
+  type: string
+EyeCameraSettings:
+  name: EyeCameraSettings
+  display_name: Eye Camera Settings
+  description: |
+    A field to store any settings that influence the resolution
+    and quality of the eye imagery.
+    Autowhitebalance? Changes in sharpness?
```

I think an example might be better than question marked phrases:

```suggestion
    A field to store any settings that influence the resolution
    and quality of the eye imagery.
    For example, `{"Autowhitebalance": true}`.
```